rsmtool/utils/logging.py

```diff
@@ -29,17 +29,30 @@ class LogFormatter(logging.Formatter):
     err_fmt = "ERROR: %(msg)s"
     dbg_fmt = "DEBUG: %(module)s: %(lineno)d: %(msg)s"
 
-    def __init__(self, fmt="%(levelno)s: %(msg)s"):  # noqa: D107
+    def __init__(self, fmt="%(levelno)s: %(msg)s"):
```

Comment:
Is a type hint missing here? 

```suggestion
    def __init__(self, fmt: str = "%(levelno)s: %(msg)s"):
```